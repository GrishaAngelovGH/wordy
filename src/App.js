import Wordy from './components/Wordy'

function App() {
  return (
    <div className="row g-0 justify-content-center text-center">
      <div className="col-3">
        <Wordy />
      </div>
    </div>
  )
}

export default App
