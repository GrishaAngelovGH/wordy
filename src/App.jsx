import Wordy from './components/Wordy'

function App() {
  return (
    <div className='row g-0 justify-content-center text-center bg-primary bg-gradient vh-100'>
      <div className='col-7 col-md-3'>
        <Wordy />
      </div>
    </div>
  )
}

export default App
