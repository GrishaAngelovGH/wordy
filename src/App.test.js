import { render } from '@testing-library/react'

import App from './App'

test('should render component', () => {
  const container = render(<App />)

  expect(container).toMatchSnapshot()
})