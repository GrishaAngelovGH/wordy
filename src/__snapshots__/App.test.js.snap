// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row g-0 justify-content-center text-center bg-primary bg-gradient vh-100"
      >
        <div
          class="col-7 col-md-3"
        >
          <div
            class="row mt-1"
          >
            <div
              class="col-md-12"
            >
              <h3
                class="alert-success rounded"
              >
                Wordy
              </h3>
              <button
                class="btn btn-light invisible"
              >
                Restart
              </button>
              <div
                class="mt-1"
              >
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
                <div
                  class="d-flex justify-content-center"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                  <div
                    class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                  />
                </div>
              </div>
              <div
                class="mt-md-3 keyboard"
              >
                <div
                  class="d-flex"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    Q
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    W
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    E
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    R
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    T
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    Y
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    U
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    I
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    O
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    P
                  </div>
                </div>
                <div
                  class="d-flex"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    A
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    S
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    D
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    F
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    G
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    H
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    J
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    K
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    L
                  </div>
                </div>
                <div
                  class="d-flex"
                >
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key "
                    role="button"
                  >
                    <svg
                      class="bi bi-arrow-return-left"
                      fill="currentColor"
                      height="16"
                      viewBox="0 0 16 16"
                      width="16"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z"
                        fill-rule="evenodd"
                      />
                    </svg>
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    Z
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    X
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    C
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    V
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    B
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    N
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                    role="button"
                  >
                    M
                  </div>
                  <div
                    class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key "
                    role="button"
                  >
                    <svg
                      class="bi bi-backspace"
                      fill="currentColor"
                      height="16"
                      viewBox="0 0 16 16"
                      width="16"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M5.83 5.146a.5.5 0 0 0 0 .708L7.975 8l-2.147 2.146a.5.5 0 0 0 .707.708l2.147-2.147 2.146 2.147a.5.5 0 0 0 .707-.708L9.39 8l2.146-2.146a.5.5 0 0 0-.707-.708L8.683 7.293 6.536 5.146a.5.5 0 0 0-.707 0z"
                      />
                      <path
                        d="M13.683 1a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2h-7.08a2 2 0 0 1-1.519-.698L.241 8.65a1 1 0 0 1 0-1.302L5.084 1.7A2 2 0 0 1 6.603 1h7.08zm-7.08 1a1 1 0 0 0-.76.35L1 8l4.844 5.65a1 1 0 0 0 .759.35h7.08a1 1 0 0 0 1-1V3a1 1 0 0 0-1-1h-7.08z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row g-0 justify-content-center text-center bg-primary bg-gradient vh-100"
    >
      <div
        class="col-7 col-md-3"
      >
        <div
          class="row mt-1"
        >
          <div
            class="col-md-12"
          >
            <h3
              class="alert-success rounded"
            >
              Wordy
            </h3>
            <button
              class="btn btn-light invisible"
            >
              Restart
            </button>
            <div
              class="mt-1"
            >
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
              <div
                class="d-flex justify-content-center"
              >
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
                <div
                  class="d-flex justify-content-center align-items-center border border-2 fw-bold rounded box bg-white text-dark"
                />
              </div>
            </div>
            <div
              class="mt-md-3 keyboard"
            >
              <div
                class="d-flex"
              >
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  Q
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  W
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  E
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  R
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  T
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  Y
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  U
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  I
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  O
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  P
                </div>
              </div>
              <div
                class="d-flex"
              >
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  A
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  S
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  D
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  F
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  G
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  H
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  J
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  K
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  L
                </div>
              </div>
              <div
                class="d-flex"
              >
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key "
                  role="button"
                >
                  <svg
                    class="bi bi-arrow-return-left"
                    fill="currentColor"
                    height="16"
                    viewBox="0 0 16 16"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  Z
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  X
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  C
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  V
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  B
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  N
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key undefined"
                  role="button"
                >
                  M
                </div>
                <div
                  class="d-flex justify-content-center align-items-center border boder-2 rounded m-1 text-white key "
                  role="button"
                >
                  <svg
                    class="bi bi-backspace"
                    fill="currentColor"
                    height="16"
                    viewBox="0 0 16 16"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M5.83 5.146a.5.5 0 0 0 0 .708L7.975 8l-2.147 2.146a.5.5 0 0 0 .707.708l2.147-2.147 2.146 2.147a.5.5 0 0 0 .707-.708L9.39 8l2.146-2.146a.5.5 0 0 0-.707-.708L8.683 7.293 6.536 5.146a.5.5 0 0 0-.707 0z"
                    />
                    <path
                      d="M13.683 1a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2h-7.08a2 2 0 0 1-1.519-.698L.241 8.65a1 1 0 0 1 0-1.302L5.084 1.7A2 2 0 0 1 6.603 1h7.08zm-7.08 1a1 1 0 0 0-.76.35L1 8l4.844 5.65a1 1 0 0 0 .759.35h7.08a1 1 0 0 0 1-1V3a1 1 0 0 0-1-1h-7.08z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
